components/BecomeASponsor.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="become-sponsor">
+    <img src="assets/gift.svg" class="become-sponsor__icon">
```

Comment:
Let's use icons from [CodeX Icons Pack](https://github.com/codex-team/icons)

Reply:
But they are different, it is opened here, in icon pack it is closed

Reply:
Ok, I see. Let's use gift icon directly and the other ones from icon pack